aws-federation/env/staging

```diff
@@ -0,0 +1 @@
+roleName=
```

Comment:
we have aws-federation lambda running in every environment in use-east region. use the execution role of it here.